src/iqm/iqm_client/iqm_client.py

```diff
@@ -593,60 +68,56 @@ class IQMClient:
             it sends to the server. The signature is appended to IQMClients own version
             information and is intended to carry additional version information,
             for example the version information of the caller.
-        tokens_file: Optional path to a tokens file used for authentication.
-            This can also be set in the IQM_TOKENS_FILE environment variable.
-            If tokens_file is set, auth_server_url, username and password
-            must not be set.
-        token: If an IQM token is long-lived and the server provides it in plain text format,
-            it can be passed in this argument. It can also be provided in the IQM_TOKEN
-            environment variable. If ``tokens_file`` (or IQM_TOKENS_FILE) is set, it will
-            override the value of this token.
 
     Keyword Args:
+        token (str): Optional long-lived IQM token in plain text format.
```

Comment:
```suggestion
        token: Long-lived IQM token in plain text format.
```
Also the other parameters need this fix.

Reply:
fixed